docs/getting-started-scalardb.md

```diff
@@ -0,0 +1,386 @@
+   ```console
+   kubectl get svc scalardb-envoy
+   ```
+   [Command execution result]
+   ```console
+   NAME             TYPE           CLUSTER-IP       EXTERNAL-IP   PORT(S)                           AGE
+   scalardb-envoy   LoadBalancer   10.103.103.188   127.0.0.1     60051:32344/TCP,50052:30921/TCP   13m
+   ```
+
+## Step 5. Start Client container
+
+
+1. Start a Client container on the `minikube` network.
+   ```console
+   docker run -d --name scalardb-client --hostname scalardb-client --network minikube --entrypoint sleep ubuntu:20.04 inf
+   ```
+
+1. Check the status of the Client container.
+   ```console
+   docker ps -f name=scalardb-client
+   ```
+   [Command execution result]
+   ```console
+   CONTAINER ID   IMAGE          COMMAND       CREATED              STATUS              PORTS     NAMES
+   a49eaf0c2442   ubuntu:20.04   "sleep inf"   About a minute ago   Up About a minute             scalardb-client
+   ```
+
+## Step 6. Run Scalar DB sample application in the Client container
+
+
+The following explains the minimum steps. If you want to know more details about Scalar DB, please refer to the [Getting Started with Scalar DB
+](https://github.com/scalar-labs/scalardb/blob/master/docs/getting-started-with-scalardb.md).
+
+1. Run bash in the Client container.
+   ```console
+   docker exec -it scalardb-client bash
+   ```
+   After this step, run each command in the Client container.  
+
+1. Install Git, OpenJDK 8, and curl in the Client container.
+   ```console
+   apt update && DEBIAN_FRONTEND="noninteractive" TZ="Etc/UTC" apt install -y git openjdk-8-jdk curl
+   ```
+
+1. Clone Scalar DB git repository.
+   ```console
+   git clone https://github.com/scalar-labs/scalardb.git
+   ```
+
+1. Change directory to `scalardb/`.
+   ```console
+   cd scalardb/
+   ```
+   ```console
+   pwd
+   ```
+   [Command execution result]
+   ```console
+   /scalardb
+   ```
+
+1. Change branch to arbitrary version.
+   ```console
+   git checkout -b v3.5.1 refs/tags/v3.5.1
+   ```
+   ```console
+   git branch
+   ```
+   [Command execution result]
+   ```console
+     master
+   * v3.5.1
+   ```
+   If you want to use another version, please specify the version (tag) you want to use.
+
+1. Build Scalar DB as a library.
+   ```console
+   ./gradlew installDist
+   ```
+
+1. Change directory to `docs/getting-started/`.
+   ```console
+   cd docs/getting-started/
+   ```
+   ```console
+   pwd
+   ```
+   [Command execution result]
+   ```console
+   /scalardb/docs/getting-started
+   ```
+
+1. Download Schema Loader from [Scalar DB Releases](https://github.com/scalar-labs/scalardb/releases).
+   ```console
+   curl -OL https://github.com/scalar-labs/scalardb/releases/download/v3.5.1/scalardb-schema-loader-3.5.1.jar
+   ```
+   You need to use the same version of Scalar DB and Schema Loader.
+
+1. Create a configuration file (scalardb-schema-loader.properties) for Schema Loader.
```

Comment:
We can create the schema via Scalar DB Server, so we can use the same configuration (`scalardb.properties`) as the one used by the sample application. What do you think?

Reply:
Thank you for your review!

I didn't know we can create the schema via Scalar DB Server...
(I thought I need to run Schema Loader against the backend DB directly...)

I agree with using the same configuration (scalardb.properties), because the document will be more simple!

I updated the document in 124b3927b742ef437d5ea4d110185d4304b2c1e3.
Please take a look.